lib/adapter/cucumber.js

```diff
@@ -66,13 +66,13 @@ const createTestomatFormatter = apiKey => {
       if (event.result.exception) {
         error = event.result.exception.message;
       }
-
-      if (event.result.status !== TRConstants.PASSED) {
+      console.log(scenario, ' : ', status);
```

Comment:
do we need this console.log, or this is debug thing?

Reply:
Yes just to list the status of the scenario. Since we use custom formmater there will not be any output. We can add chalk and make it good. But it should be sufficient for now

Reply:
Ok, makes sense. Got it!